pkg/registry/servicecatalog/binding/storage.go

```diff
@@ -112,45 +110,22 @@ func GetAttrs(obj runtime.Object) (labels.Set, fields.Set, bool, error) {
 
 // NewStorage creates a new rest.Storage responsible for accessing ServiceBinding
 // resources
-func NewStorage(opts server.Options) (rest.Storage, rest.Storage, error) {
-	prefix := "/" + opts.ResourcePrefix()
-
-	storageInterface, dFunc := opts.GetStorage(
-		&servicecatalog.ServiceBinding{},
-		prefix,
-		bindingRESTStrategies,
-		NewList,
-		nil,
-		storage.NoTriggerPublisher,
-	)
-
+func NewStorage(optsGetter generic.RESTOptionsGetter) (serviceBindings, serviceBindingStatus rest.Storage, err error) {
 	store := registry.Store{
-		NewFunc: EmptyObject,
-		// NewListFunc returns an object capable of storing results of an etcd list.
-		NewListFunc: NewList,
-		KeyRootFunc: opts.KeyRootFunc(),
-		KeyFunc:     opts.KeyFunc(true),
-		// Retrieve the name field of the resource.
-		ObjectNameFunc: func(obj runtime.Object) (string, error) {
-			return scmeta.GetAccessor().Name(obj)
-		},
-		// Used to match objects based on labels/fields for list.
-		PredicateFunc: Match,
-		// DefaultQualifiedResource should always be plural
+		NewFunc:                  func() runtime.Object { return &servicecatalog.ServiceBinding{} },
```

Comment:
Why did you stop using `EmptyObject`? I didn't see you remove the definition of `EmptyObject` and this change wasn't part of what the PR set out to do.

Reply:
Oops sorry - this was due to copy/paste/tweak code from other projects. Will switch back to `EmptyObject`.

Reply:
Done.